pkg/kubectl/pdb.go

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
Thinking out loud, with the number of generators we should have a separate package for them, created #36805 to discuss the idea further.